nexus/db-model/src/rack.rs

```diff
@@ -28,6 +29,22 @@ impl Rack {
             rack_subnet: None,
         }
     }
+
```

Comment:
(can't put this comment in the right place, which is L20)

I think maybe `rack_subnet` be non-`pub` if we want people to use the new getter.

Reply:
The app code for setting the field when syncing the bootstore uses the field directly, and that seems correct to me.